EXTCAP.md

```diff
@@ -0,0 +1,180 @@
+# Spinel Extcap Reference
+The Spinel Extcap provides a more user-friendly way to use OpenThread Sniffer. Spinel 
+Extcap is primarily targeted for integrating OpenThread Sniffer with Wiresahrk,  and 
+is suitable for user to use OpenThread Sniffer by Wireshark GUI.
+
+## System Requirements
+
+The tool has been tested on the following platforms:
+
+| Platforms  | Version          |
+|------------|------------------|
+| Ubuntu     | 14.04 Trusty     |
+| Mac OS     | 10.11 El Capitan |
+| Windows 10 | 1803             |
+
+| Language  | Version          |
+|-----------|------------------|
+| Python    | 2.7.10           |
+
+| Software  | Version          |
+|-----------|------------------|
+| Wireshark | 3.0.3            |
+
+### Package Installation
+#### Automatic installation from source
```

Comment:
Add blank lines before and after all header lines and lists throughout the document for source readability for future editors